controller/instance_manager_controller.go

```diff
@@ -1256,6 +1272,7 @@ func (imc *InstanceManagerController) enqueueSettingChange(obj interface{}) {
 
 func (imc *InstanceManagerController) cleanupInstanceManagerPod(imName string) error {
 	imc.stopMonitoring(imName)
+	imc.stopBackingImageMonitoring(imName)
```

Comment:
Should we check if it v2 data engine before stop backing image monitoring?
```
if types.IsDataEngineV2(im.Spec.DataEngine) {
   imc.stopBackingImageMonitoring(....
}
```

Reply:
No I don't think so
in `stopBackingImageMonitoring`
we return if we can't find the key in the map
```
       backingImageMonitorName := types.GetBackingImageMonitorName(imName)
	stopCh, ok := imc.instanceManagerMonitorMap[backingImageMonitorName]
	if !ok {
		return
	}
```